code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -3577,7 +3577,17 @@
 				if(prob(10))
 					to_chat(H,"<span class='warning'>You really don't feel very good.</span>")
 				if(prob(5))
-					H.adjustToxLoss(0.1)
+					switch(pick(1,2,3,4,5))
+						if(1)
+							H.adjustToxLoss(0.5)
+						if(2)
+							H.adjustToxLoss(0.5)
+						if(3)
+							H.adjustToxLoss(0.5)
+						if(4)
+							H.adjustToxLoss(0.5)
+						if(5)
+							H.adjustToxLoss(0.5)
```

Comment:
You sure about that random type thing?

Also holy shit you quintupled the damage